webui/react/src/services/apiConfig.ts

```diff
@@ -944,16 +944,18 @@ export const timeSeries: DetApi<
   postProcess: (response: Api.V1CompareTrialsResponse) => {
     return response.trials.map(decoder.decodeTrialSummary);
   },
-  request: (params: Service.TimeSeriesParams) =>
-    detApi.Experiments.compareTrials(
+  request: (params: Service.TimeSeriesParams) => {
+    return detApi.Experiments.compareTrials(
       params.trialIds,
       params.maxDatapoints,
-      params.metricNames.map((m) => m.name),
+      undefined,
       params.startBatches,
       params.endBatches,
       params.metricType ? Type.metricTypeParamMap[params.metricType] : 'METRIC_TYPE_UNSPECIFIED',
       undefined,
-    ),
+      params.metrics.map((metric) => [metric.group, metric.name].join('.')),
```

Comment:
this change uses a completely new param of `metricIds` instead of `metricNames` (left as `undefined`) in order to return time series data for generic group metrics